src/ui/assets/AssetsUpload.js

```diff
@@ -53,6 +54,7 @@ const AssetsUpload = ({
       </div>
       <UploadAssetModalContainer
         onAssetSelected={onAssetSelected}
+        lockGroup={ownerGroup}
```

Comment:
@jeffgo10 why are we using lockGroup instead of ownerGroup?

Reply:
I meant the parameter to be locked on a certain group but yeah you're right, technically we should make it ownerGroup